feature/experimental/system/gnmi/benchmarking/ate_tests/drained_configuration_convergence_time/drained_configuration_convergence_time_bgp_test.go

```diff
@@ -0,0 +1,264 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// drained_configuration_convergence_time_test is used to verify bgp test scenarios
+// as given in gnmi1.3 testcase.
+package drained_configuration_convergence_time_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/openconfig/featureprofiles/feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+const (
+	asPathRepeatValue      = 3
+	aclStatement2          = "20"
+	aclStatement3          = "30"
+	setAspathPrependPolicy = "SET-ASPATH-PREPEND"
+	setMedPolicy           = "SET-MED"
+	bgpMed                 = 25
+)
+
+// setMED is used to configure routing policy to set BGP MED on DUT
+// TODO : SetMED is not supported: https://github.com/openconfig/featureprofiles/issues/759
+func setMED(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+
+	// Clear existing routing policies on DUT
+	gnmi.Delete(t, dut, gnmi.OC().RoutingPolicy().Config())
+
+	// Configure SetMED on DUT
+	d := &oc.Root{}
+	rp := d.GetOrCreateRoutingPolicy()
+	pdef5 := rp.GetOrCreatePolicyDefinition(setMedPolicy)
+	actions5 := pdef5.GetOrCreateStatement(aclStatement3).GetOrCreateActions()
+	// TODO : SetMED is not supported: https://github.com/openconfig/featureprofiles/issues/759
```

Comment:
Can you standardize all the code blocks that are commented throughout to this form:

```
// TODO: Uncomment when SetMED is supported.
// https://github.com/openconfig/featureprofiles/issues/759
// [commented out code here]
```

Reply:
I have updated this for unsupported block code , please check latest code.